src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+  const dayRentCost = 40;
+  const thirdDayDiscount = 20;
+  const seventhDayDiscount = 50;
+
+  if (days < 3) {
+    total = days * dayRentCost;
+  }
+
+  if (days >= 3) {
+    total = (days * dayRentCost) - thirdDayDiscount;
+  }
+
+  if (days >= 7) {
+    total = (days * dayRentCost) - seventhDayDiscount;
+  }
```

Comment:
```suggestion
  if (days < 3) {
    return days * dayRentCost;
  }

  if (days < 7) {
    return (days * dayRentCost) - thirdDayDiscount;
  }
  
  return (days * dayRentCost) - seventhDayDiscount;
```
3 и 7 тоже вынеси в переменные